docs/api/index.md

```diff
@@ -0,0 +1,53 @@
+# API Documentation
+
+Your own tools can interact with OpenCVE using its REST API. You can list the last CVEs per project, filter them by vendor or CVSS.
+
+The API of the OpenCVE Cloud instance is available on [https://app.opencve.io/api/](https://www.opencve.io/api).
```

Comment:
Use https://app.opencve.io/api/ for both as the main entrypoint (https://www.opencve.io/api is redirected to app for compliance with v1)